deployment/artifacts/lynx.json

```diff
@@ -2304,12 +2304,68 @@
                         {
                             "name": "_coordinator",
                             "type": "address",
-                            "internalType": "contract Coordinator"
+                            "components": null,
+                            "internal_type": "contract Coordinator"
                         },
                         {
-                            "name": "_executor",
+                            "name": "_accessController",
                             "type": "address",
-                            "internalType": "address"
+                            "components": null,
+                            "internal_type": "contract GlobalAllowList"
```

Comment:
We need to be aware of this change in outputs. Thus far, I've been manually changing them back to the old values i.e.:
1. find and replace `internal_type` -> `internalType`
2. Removing `"components": null,` lines

I haven't delved into whether there are compatibility issues with the nodes using the updated values in the abi with web3py. Been putting it off really.

@cygnusv filed an issue some time ago with the ape folks - https://github.com/ApeWorX/ethpm-types/issues/116 - but there doesn't seem to be any updates since then. I added a fresh comment - https://github.com/ApeWorX/ethpm-types/issues/116#issuecomment-2214460891.

Reply:
This is handled in #286 .